### DISCUSSION TITLE
Mod "Open pinned items with double click" ?

### DISCUSSION THREAD

**Discussion by sypqys** [ramensoftware/windhawk-mods#467] [Q&A] [answered]
Hi !

it is active but it don't work...
and Taskbar height and size icon also...
![Eac2J6Bagb](https://github.com/ramensoftware/windhawk-mods/assets/27967268/c716d9fc-6625-4c05-9828-074e522cedc8)

What happens ?

Thanks

**Comment by m417z** [comment, marked as answer]
It should work now. Please disable and re-enable the mod, or restart Windhawk.

The issue is similar to #464. As with [one of the previous updates](https://github.com/ramensoftware/windhawk-mods/issues/318#issuecomment-1741725866):

> Most taskbar-related mods didn't work on the day the update was published because it took time for Microsoft to publish debug symbols for it. Once published, the mods began working, no update to the mods was required.
> [...]
> I'm closing this issue. If there are any other incompatibilities or problems, please create a new issue.

**Reply by sypqys** [reply]
Yes it works disabling and reenabling the mod ! Thanks
